packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketVerify/CoinmarketVerifyOptions.tsx

```diff
@@ -12,23 +14,40 @@ export const CoinmarketVerifyOptions = ({
     selectedAccountOption,
     isMenuOpen,
     onChangeAccount,
-}: CoinmarketVerifyOptionsProps) => (
-    <Select
-        onChange={(selected: CoinmarketVerifyFormAccountOptionProps) => onChangeAccount(selected)}
-        value={selectedAccountOption}
-        isClearable={false}
-        options={selectAccountOptions}
-        minValueWidth="70px"
-        formatOptionLabel={option => (
-            <CoinmarketVerifyOptionsItem option={option} receiveNetwork={receiveNetwork} />
-        )}
-        menuIsOpen={isMenuOpen}
-        isDisabled={selectAccountOptions.length === 1}
-        placeholder={
-            <Translation
-                id="TR_EXCHANGE_SELECT_RECEIVE_ACCOUNT"
-                values={{ symbol: receiveNetwork?.toUpperCase() }}
-            />
-        }
-    />
-);
+}: CoinmarketVerifyOptionsProps) => {
+    const { cryptoIdToPlatformName, cryptoIdToCoinName } = useCoinmarketInfo();
+
+    if (!receiveNetwork) {
+        return null;
+    }
```

Comment:
I'd make `receiveNetwork` prop non-nullable and remove this `if`.
`CoinmarketVerifyOptions` is called from two places, and `receiveNetwork` is non-nullable at both, so there seems to be no reason to keep it nullable (and potentially cause problems in future).

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/14294/commits/8835cb5273c4bb999ec1a3e73a4c86241fdfa730